content/patterns/disclosure/examples/disclosure-faq.html

```diff
@@ -179,11 +180,11 @@ <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
               </td>
               <td>
                 <ul>
-                  <li>Indicates that the container controlled by the disclosure button is <em>hidden</em>.</li>
+                  <li>ディスクロージャーボタンによって制御されるコンテナが<em>非表示</em>であることを示します。</li>
                   <li>
-                    CSS attribute selectors (e.g. <code>[aria-expanded="false"]</code>) are used to synchronize the visual states with the value of the <code>aria-expanded</code> attribute.
+                    CSS 属性セレクター（例：<code>[aria-expanded="false"]</code>）は、視覚的な状態を <code>aria-expanded</code> 属性の値と同期させるために使用されます。
                   </li>
-                  <li>The visual indicator of the show/hide state is created using CSS <code>:before</code> pseudo element and <code>content</code> property so the image is reliably rendered in high contrast mode of operating systems and browsers.</li>
+                  <li>表示/非表示状態の視覚的インジケーターは、CSS <code>:before</code> 疑似要素と <code>content</code> プロパティを使用して作成されるため、画像はオペレーティングシステムとブラウザのハイコントラストモードで確実にレンダリングされます。</li>
```

Comment:
imageはディスクロージャーの三角のことで、日本語で「画像」というと少し違和感があるのですが、適当な訳も見当たらず……
ひとまずtheを訳出した方がわかりやすそうです
```suggestion
                  <li>表示/非表示状態の視覚的インジケーターは、CSS <code>:before</code> 疑似要素と <code>content</code> プロパティを使用して作成されるため、その画像はオペレーティングシステムとブラウザのハイコントラストモードで確実にレンダリングされます。</li>
```